dotnet/src/Planners/Planners.Core/Handlebars/IHandlebarsPlanner.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.Diagnostics;
+
+namespace Microsoft.SemanticKernel.Planners.Handlebars;
+
+/// <summary>
+/// Interface for planner that uses a set of semantic functions to create a sequential plan.
+/// </summary>
+public interface IHandlebarsPlanner
```

Comment:
Are we expecting multiple different implementations?

Feels like should have an `IPlanner` interface which uses generics. I think we should look at this before we make planners v1.0.

Reply:
You're right, this is really only good for the Stopwatch if we want to leverage that functionality across different planners. I removed this.